monitoring/README.md

```diff
@@ -0,0 +1,27 @@
+# Monitoring Stack for EKS
+
+This module sets up Grafana, Prometheus, Loki and Promtail stack.
+This is all you need to monitor your cluster.
+
+## Usage:
+
+```hcl
+module "monitoring" {
+  source                            = "github.com/contiamo/terraform//monitoring"
```

Comment:
should this module really be `eks-monitoring` because some of the settings and assumptions are _very_ AWS specific

Reply:
It doesn't matter really. It's up to the user to call their modules whatever they want.

Reply:
ok sure, but don't yell at me when you go looking for generic monitoring stuff and grab this